test/framework/vsphere.go

```diff
@@ -31,6 +30,7 @@ const (
 	vsphereTlsThumbprintVar     = "T_VSPHERE_TLS_THUMBPRINT"
 	vsphereUsernameVar          = "EKSA_VSPHERE_USERNAME"
 	vspherePasswordVar          = "EKSA_VSPHERE_PASSWORD"
+	VsphereClusterIPPoolEnvVar  = "T_VSPHERE_CLUSTER_IP_POOL"
```

Comment:
Renamed this back to vSphere specific as provider tests run on different networks so this can't be a shared property as I originally thought